### ISSUE TITLE
Method name 'select' ERROR

### ISSUE THREAD

**Issue by masuhara** [rolandleth/LTHRadioButton#2]
Hi, I'm using LTHRadioButton with CocoaPods. When I was using this library, I met an error.
The name of method 'select' may be not good.

Xcode8.1
Swift3.0.1

<img width="1401" alt="2016-12-06 14 42 09" src="https://cloud.githubusercontent.com/assets/1835427/20914382/a5154cec-bbc2-11e6-9e11-270c816ac411.png">

**Comment by rolandleth** [comment]
Hey,


Sadly, I don't fully understand why that happened, since the method is actually `select(animated: Bool)`. I created a new Objective-C project, I added the file, it asked me to convert to latest Swift syntax, and the file ended up the same as yours, but that's as if it converted from Swift 2.3 to 3.0, which is not the case.

But I went in the file, undo-ed the changes the converter did, and now it's properly exposed as `selectWithAnimated:` - which doesn't sound that great, maybe `selectWithAnimation:` / `selectAnimated:` would be better?
